src/OpenTelemetry.AutoInstrumentation.Native/util.cpp

```diff
@@ -97,6 +99,53 @@ std::vector<WSTRING> GetEnvironmentValues(const WSTRING& name)
     return GetEnvironmentValues(name, L',');
 }
 
+
+std::vector<WSTRING> GetEnvironmentVariables(const std::vector<WSTRING> &prefixes)
+{
+    std::vector<WSTRING> env_strings;
+#ifdef _WIN32
+    auto env_variables = GetEnvironmentStrings();
```

Comment:
The remark linked below seems very useful to understand what is going on here: https://learn.microsoft.com/en-us/windows/win32/api/processenv/nf-processenv-getenvironmentstrings#remarks

Reply:
Like done for non-Windows this should also be const.

Reply:
Both fixed: 93f0408db630f7682e1f27a9235a2dc85cb83a0d